### PR TITLE
Add a 'push' and 'commit' options to the freeze and release script.

### PR DESCRIPTION
**Pull request by jansiegel**
### Context
This PR adds a `--commit` and `--push` options to the freeze script and a `--push` option to the release scripts, and sets them all to `false` by default.

<sup>[skip changelog]</sup>

### How has this been tested?
Tested manually.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or improvement (non-breaking change which adds functionality) **(internal change, doesn't require bumping the minor version number)**
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Affected project(s):
- [x] `handsontable`
- [x] `@handsontable/angular`
- [x] `@handsontable/react`
- [x] `@handsontable/vue`

### Checklist:

- [x] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md) and I confirm that my code follows the code style of this project.
- [x] I have signed the [Contributor License Agreement](https://docs.google.com/forms/d/e/1FAIpQLScpMq4swMelvw3-onxC8Jl29m0fVp5hpf7d1yQVklqVjGjWGA/viewform?c=0&w=1)
- [ ] My change requires a change to the documentation.

### CODE REVIEW

File: scripts/freeze.mjs [hunk 1]
```diff
@@ -17,6 +19,17 @@ import {
...
+  .boolean('commit')
+  .default('commit', false)
+  .describe('commit', '`true` if the state of the repo should be committed automatically at the end of the freeze' +
+    ' process.')
+  .boolean('push')
+  .default('push', false)
+  .describe('push', '`true` if the state of the repo should be pushed to remote automatically at the end of the' +
+    ' freeze process.')
+  .argv;
+
```
Comment (budnix) [inline scripts/freeze.mjs:32]:
```suggestion
const argv = yargs(hideBin(process.argv))
  .boolean('commit')
  .default('commit', false)
  .describe('commit', '`true` if the state of the repo should be committed automatically at the end of the freeze' +
    ' process.')
  .boolean('push')
  .default('push', false)
  .describe('push', '`true` if the state of the repo should be pushed to remote automatically at the end of the' +
    ' freeze process.')
  .argv;

const [version, releaseDate] = argv._;
```
Let's the `yargs` manage all CLI arguments. Under the `_` key there are non-hyphened options (https://github.com/yargs/yargs/blob/master/docs/examples.md#and-non-hyphenated-options-too-just-use-argv_)

Comment (jansiegel) [reply scripts/freeze.mjs:32]: Ah, I was looking for it but didn't find it 👌

**Review by budnix — DISMISSED** [review]
👍

**Review by aninde — DISMISSED** [review]
I approve as it does not affect the size of the package and it's very useful.

**Review by budnix — APPROVED** [review]

**Review by aninde — APPROVED** [review]

**Pull request merged**